### ISSUE TITLE
Reorder action numbers

### ISSUE THREAD

**Issue by Makman2** [coala-analyzer/coala#271]
And next issue for coala: I tried the SpaceConsistencyBear and my options are like these:

1: Open the file in an editor.
2: Apply the patch automatically.
0: No action.

No action should be on top of the listing, 1 2 and then 0 is quite confusing :P

**Comment by sils1297** [comment]
How about:
```
|    |    | The following options are applicable to this result (choose 0 for no action):
|    |    |  1: Open the file in an editor.
|    |    | Please enter the number of the action you want to execute.
```

**Comment by sils1297** [comment]
will be fixed with this commit if you don't have anything against this variant.

**Comment by Makman2** [comment]
Seems good for me. Especially there you see that you can type `0` everytime to perform no action.

**Comment by sils1297** [comment]
Cool. Closing as soon as this gets merged. Care for a review? I've fixed two of the bugs you reported within ten minutes, now's your turn ;) #261 contains the fixes, #264 contains a cool new tutorial (if you didn't check it out yet, please do, there's much love in it I promise.)

**Comment by Makman2** [comment]
Yep runs :D
